core/error.rs

```diff
@@ -354,15 +354,12 @@ impl JsError {
     {
       let state = JsRuntime::state_from(scope);
       let mut source_mapper = state.source_mapper.borrow_mut();
-      for (i, frame) in frames.iter().enumerate() {
+      if let Some(frame) = frames.first() {
         if let (Some(file_name), Some(line_number)) =
           (&frame.file_name, frame.line_number)
         {
-          if !file_name.trim_start_matches('[').starts_with("ext:") {
-            source_line = source_mapper.get_source_line(file_name, line_number);
-            source_line_frame_index = Some(i);
-            break;
-          }
```

Comment:
Verify this in a separate PR. We support source maps for `ext:` modules so most likely this is not needed